src/history/hs.c

```diff
@@ -685,7 +685,12 @@ __wt_hs_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MUL
             WT_ASSERT(session, upd->type == WT_UPDATE_STANDARD || upd->type == WT_UPDATE_MODIFY);
 
             __wt_modify_vector_pop(&modifies, &prev_upd);
-            stop_ts_pair.timestamp = prev_upd->start_ts;
+
+            /* Set the stop timestamp from durable or commit timestamp. */
+            if (prev_upd->durable_ts != WT_TS_NONE)
```

Comment:
Does durable_ts will not be equal to commit_ts, if durable_ts is not explicitly set ? I prefer not to have this check with WT_TS_NONE, if we later change the default initialization value, we could miss to change this check, which will lead to issues. Please try to remove the check or update the comments to indicate we are comparing with default init value.

Reply:
Already there exists code that sets the durable timestamp same as to commit timestamp. So I removed the check against durable timestamp and assign it directly.